src/fairchem/core/trainers/base_trainer.py

```diff
@@ -357,15 +361,122 @@ def load_datasets(self) -> None:
             )
 
     def load_task(self):
-        # Normalizer for the dataset.
-        normalizer = self.config["dataset"].get("transforms", {}).get("normalizer", {})
+        # load or fit element references for dataset
+        elementrefs = (
+            self.config["dataset"].get("transforms", {}).get("element_references", {})
```

Comment:
nit; would it make sense to break this out into two other class methods, load_references(), load_normalizers()?

Reply:
I put it into a single new method `load_references_and_normalizers` to ensure that element references are always done before normalizers.

Reply:
+1 to what Misko said. To address your concerns we can do :

```
def load_target_transforms():
     load_references()
     load_normalizer()

def load_references():
    #blah blah

def load_normalizers():
   # blah blah
```

nit - can we move this method under `load_task`.

A